ant-networking/src/metrics/mod.rs

```diff
@@ -42,9 +41,14 @@ pub(crate) struct NetworkMetricsRecorder {
 
     // metrics from ant-networking
     pub(crate) connected_peers: Gauge,
+    pub(crate) connected_relay_clients: Gauge,
     pub(crate) estimated_network_size: Gauge,
+    pub(crate) percentage_of_relay_peers: Gauge<f64, AtomicU64>,
     pub(crate) open_connections: Gauge,
     pub(crate) peers_in_routing_table: Gauge,
+    pub(crate) relay_peers_in_routing_table: Gauge,
+    pub(crate) peers_in_non_full_buckets: Gauge,
+    pub(crate) relay_peers_in_non_full_buckets: Gauge,
```

Comment:
`peers_in_non_full_buckets` and `relay_peers_in_non_full_buckets` may can be removed before merge